testsuite/dockerfiles/server-all-in-one-dev/Dockerfile

```diff
@@ -14,10 +13,11 @@ RUN zypper -n --gpg-auto-import-keys ref && \
       cpio \
       spacecmd \
       expect \
-      prometheus \
+      hostname \
       wget && \
```

Comment:
Take into account that this might collide with some changes from Cedric, about getting rid of expect and wget.

Reply:
I know, this is why I am installing them. Once expect and wget are gone, we can remove them from this dockerfile.

Reply:
why are you adding `hostname` in that container?

Reply:
We need hostname for the tests to work. The "remote node" class initialization of our testsuite uses that under the hood.

https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/support/remote_node.rb#L31

Reply:
Be aware that the released container image will have no repo at some point soon. Installing `hostname` inside the container seems really hacky to me.

Reply:
Couldn't we hack `remote_node.rb` to set the hostname to a fixed value if `hostname` tool is not available?

Reply:
We only call the hostname at Host layer Cedric. But here, on GH environment, we kind of mix both host and container in one.

Reply:
fixed in the last commit